projects/ngqp/core/src/lib/directives/query-param.directive.ts

```diff
@@ -0,0 +1,64 @@
+import { Directive, Inject, Input, OnChanges, OnDestroy, Optional, Self, SimpleChanges } from '@angular/core';
+import { QueryParamGroupService } from './query-param-group.service';
+import { QueryParamController } from './query-param-controller.interface';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { selectValueAccessor } from '../accessors/util';
+import { QueryParam } from '../model/query-param';
+import { QueryParamGroup } from '../model/query-param-group';
+
+/**
+ * Binds a standalone {@link QueryParam} to a DOM element.
+ *
+ * This directive accepts the name of a {@link QueryParam} without having outer {@link QueryParamGroup}.
```

Comment:
It doesn't accept the name of one, it accepts the object itself, so I think this should read "This directive accepts a {@link QueryParam} …"

Reply:
Fixed